examples/config-many-files/teams-enroll-secret.yml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: v1
+kind: enroll_secret
+spec:
+  secrets:
+  - active: true
+    name: "default"
```

Comment:
Will `name` of secret be something that can be targeted by queries?

Currently, you can target (non-exhaustive list):
* individual host
* hosts by labels

I imagine in the new world of teams, it might be useful to support querying by `team`, querying by `secret name`, and querying by the current query targeting options.

Reply:
No, we're not planning on supporting querying by `name`.

Querying by `team` will be supported. 

We don't plan to support querying by `name` because the working hypothesis is that teams will be used primarily for host segmentation. 

Multiple enroll secrets can correspond to the same team, however, query targeting will be limited to `team`, `host`, and `label`

Reply:
Targeting suggestions:
* intersection / union of different combinations of team / host / label

I imagine wanting to do something like: run this query on anything *except* this one specific team. That might be possible with labels, but only if there's a way to treat 'team' membership as part of label definition.